pkg/projects/packages.builds

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <Import Project="..\dir.props" />
```

Comment:
It was because the new model attempts to add a project reference to the pkgproj for each builds file.  We don't hit this in other repo's because it's not all crammed into the same area.  I can clean it up.